call-in-sick/README.md

```diff
@@ -0,0 +1,120 @@
+    Name it "Call in Sick", select a Billing Account if prompted, and
+    click **CREATE**.
+1.  When the project creation is complete a notification appears in the
+    upper-right of the page. Click on the **Create Project: Call in Sick** entry
+    to open the project.
+1.  Open the [**OAuth consent screen**][consent_screen] settings page for the
+    project.
+1.  In the field **Application name** enter "Attendance Bot" and click the
+    **Save** button at the bottom.
+1.  Open the [**Gmail API**][library_gmail] page in the API library and click
+    the **ENABLE** button.
+1.  Open the [**Project settings**][project_settings] page for the project.
+1.  Copy the value listed under **Project number**.
+
+Next, setup your script to use the new project and deploy it:
+
+1.  Back in the App Script Code Editor, click the menu item
+    **Resources > Cloud Platform project**.
+1.  Enter the project number into the text box and click **Set Project**. When
+    prompted, click Confirm.
+1.  When complete, click the **Close** button or **X** icon to dismiss the
+    dialog.
+1.  Click the menu item **Publish > Deploy from manifest**.
+1.  Next to the entry **Latest Version (Head)** click **Get ID**.
+1.  Select and copy the **Deployment ID** value.
+1.  Close the dialog and the **Deployments** window.
+
+Then configure the Hangouts Chat API to create your bot:
+
+1.  Back in the Google Cloud Developers Console, open the
+    [**Hangouts Chat API**][library_chat] page in the API library and click the
+    **ENABLE** button.
+1.  Once the API is enabled, on click the **Configuration** tab.
+1.  In the Configuration tab, do the following:
+    1.  In the **Bot name** box, enter "Attendance Bot".
+    1.  In the **Avatar URL box**, enter `https://goo.gl/kv2ENA`.
+    1.  In the **Description box**, enter "Call in sick with a bot".
+    1.  Under **Functionality**, select **Bot works in direct messages**.
+    1.  Under **Connection settings**, select **Apps Script project** and paste
+        your script's **Deployment ID** into the text box.
+    1.  Under **Permissions**, select **Specific people and group in your
+        domain**. In the text box under the drop-down menu, enter your email
+        address.
+    1.  Click Save changes.
+1.  After you save your changes, verify that the status on the Hangouts Chat API
+    page shows the Bot Status to be **LIVE – available to users**.
+
+Finally, test that your bot is working:
+
+1.  Open [Hangouts Chat][hangouts_chat].
+1.  Click **Find people, rooms, bots > Message a bot**.
+1.  From the list, select the **Attendance Bot** that you created.
+1.  Send the message "I'm sick" to the bot.
+1.  Click the **CONFIGURE** button and authorize access to your account.
+1.  In the resulting card, click the **SET VACATION IN GMAIL** and
+    **BLOCK OUT DAY IN CALENDAR** buttons.
+
+[code]: https://script.google.com/d/1pbuGhMkTyqfeR30QfbMzsA21qM2p0HmnStGpZUKj-QLvH0BL73UbTlSq/edit
```

Comment:
Should this be /copy and we can have the users make a copy of the script in their hub?

Reply:
As mentioned offline, this doesn't work for scripts.